.storybook/config.js

```diff
@@ -77,12 +77,12 @@ wp.i18n.sprintf = sprintf || {};
 global.wp = global.wp || wp;
 global.React = React;
 global.lodash = lodash;
-global.googlesitekit = global.googlesitekit || googlesitekit;
-global.googlesitekit.setup = global.googlesitekit.setup || googlesitekit.setup;
-global.googlesitekit.admin = global.googlesitekit.admin || googlesitekit.admin;
-global.googlesitekit.modules = global.googlesitekit.modules || googlesitekit.modules;
-global.googlesitekit.admin.assetsRoot = './assets/';
-global.googlesitekit.isStorybook = true;
+global._googlesitekitLegacyData = global._googlesitekitLegacyData || googlesitekit;
+global._googlesitekitLegacyData.setup = global._googlesitekitLegacyData.setup || googlesitekit.setup;
+global._googlesitekitLegacyData.admin = global._googlesitekitLegacyData.admin || googlesitekit.admin;
+global._googlesitekitLegacyData.modules = global._googlesitekitLegacyData.modules || googlesitekit.modules;
```

Comment:
Let's see if we can simplify this a little bit as most of this looks unnecessary.

Can you try removing the above:
```diff
- // Default Data.
- const googlesitekit = dashboardData;
```
and assigning here?
```suggestion
global._googlesitekitLegacyData = dashboardData;
```

Reply:
All updated as requested